src/Knub.ts

```diff
@@ -130,19 +131,23 @@ export class Knub<
 
       sendErrorMessageFn(channel, body) {
         void channel.send({
-          embed: {
-            description: body,
-            color: parseInt("ee4400", 16),
-          },
+          embeds: [
+            {
+              description: body,
+              color: parseInt("ee4400", 16),
```

Comment:
```suggestion
              color: 0xee4400,
```

Reply:
I assume this works properly in djs? This was probably put in by Dragory back in the day because eris didn't accept plain hex.

Reply:
`parseInt("ee4400", 16)` is equivalent to `0xee4400` 😅